data/sitesEN.json

```diff
@@ -6424,5 +6424,24 @@
     "destination_main_page": "Main_Page",
     "destination_search_path": "/w/index.php",
     "destination_content_path": "/wiki/"
+  },
+  {
+    "id": "en-bug-fables",
+    "origins_label": "Bug Fables Fandom Wiki",
+    "origins": [
+      {
+        "origin": "Bug Fables Fandom Wiki",
+        "origin_base_url": "bugfables.fandom.com",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "Bug_Fables_Wiki"
+      }
+    ],
+    "destination": "Bug Fables Wiki",
+    "destination_base_url": "bugfables.miraheze.org",
+    "destination_platform": "mediawiki",
+    "destination_icon": "bugfables.png",
+    "destination_main_page": "Bug_Fables_Wiki",
+    "destination_search_path": "/w/index.php",
+    "destination_content_path": "/wiki/"
```

Comment:
Please add a `miraheze` tag (see Adventure Zone Wiki for an example: https://github.com/KevinPayravi/indie-wiki-buddy/blob/main/data/sitesEN.json#L86).

```
"tags": [
  "miraheze"
]
```

Reply:
done